TestClass.java

```diff
@@ -0,0 +1,18 @@
+package exceptions;
+
+public class TestClass implements AutoCloseable {
+
+    public TestClass() {
+        throw new RuntimeException("Runtime Exception in TestClass");

```

Comment:
We don't need here because the idea of class that implement AutoClosable is that close method will be called automatically in try-with resources statement

Reply:
Also this exception prevents us to see that close method is called automatically in try-with resources statement.
Try to remove it and see the message from close message.